wgsl/index.bs

```diff
@@ -7861,24 +7876,6 @@ Note: The [=attribute/const=] attribute cannot be applied to user-declared funct
     function parameter.
     * In particular, an argument that is a pointer [=shader-creation error|must=] agree with the formal parameter
         on [=address space=], [=store type=], and [=access mode=].
-* For [=user-defined functions=], a parameter of pointer type [=shader-creation error|must=] be in one of
-    the following address spaces:
-    * [=address spaces/function=]
-    * [=address spaces/private=]
-* For [=built-in functions=], a parameter of pointer type [=shader-creation error|must=] be in one of
```

Comment:
I think I understand the logic of not needing to mention this in the unrestricted_pointer_parameters changelog.

I guess it's because the restriction on built-in functions is already implicitly enforced by the allowed signatures.
So in theory we could have removed this clause form the original text even without any new feature.

If I've gotten that wrong, please let me know.

Reply:
Right we had this there to avoid confusion that user-defined functions had restrictions that were listed.